test/e2e/porting_workloads_test.go

```diff
@@ -44,6 +44,8 @@ var _ = ginkgo.Describe("porting workloads testing", func() {
 				"Creating deployment(%s/%s) on the member cluster %s first to simulate a scenario where the target cluster already has a deployment with the same name",
 				deploymentNamespace, deploymentName, member1,
 			)
+			// We just need to make sure that `testNamespace` exists on the member cluster, no need to check for errors
+			member1Client.CoreV1().Namespaces().Create(context.TODO(), helper.NewNamespace(testNamespace), metav1.CreateOptions{}) // nolint
```

Comment:
What does this `// nolint` comment mean? Record for there is an unhandled err?

Reply:
Yes, otherwise the static check will throw an error:

```
test/e2e/porting_workloads_test.go:48:46: Error return value of `` is not checked (errcheck)
			member1Client.CoreV1().Namespaces().Create(context.TODO(), helper.NewNamespace(testNamespace), metav1.CreateOptions{})
```

Reply:
> We just need to make sure that `testNamespace` exists on the member cluster, no need to check for errors

Why? Seems weird.

Reply:
thanks, got it.

Reply:
> Why? Seems weird.

Because in some cases the namespace will be created in other cases first, we will get an error if we create it again, but we can ignore that, as long as the namespace exists, we can continue to create the deployment

Reply:
I know, but the namespace should be propagated to member cluster by `namespace` controller, it'll be a conflict if we create ns here.

Reply:
Why not wait for the ns ready?

Reply:
If we need to wait ns ready, we can check like this:
```
gomega.Eventually(func() bool {
	_, err := member1Client.CoreV1().Namespaces().Get(context.TODO(), testNamespace, metav1.GetOptions{})
	if err != nil {
		return false
	}
	return true
}, pollTimeout, pollInterval).Should(gomega.Equal(true))
```

Reply:
> Why not wait for the ns ready?

Reasonable, I have updated